projects/CarouselExperiments/app/src/main/res/layout/demo_010_carousel.xml

```diff
@@ -85,6 +85,7 @@
         app:carousel_backwardTransition="@+id/backward"
         app:carousel_previousState="@+id/previous"
         app:carousel_nextState="@+id/next"
+        app:carousel_infinite="true"
```

Comment:
we probably don't want to have all examples using infinite?

Reply:
I have left four infinite examples, and three normals.

Reply:
I have left four infinite examples, and three normals.